core/src/test/java/org/jdbi/v3/core/argument/TestInetAddressH2.java

```diff
@@ -29,6 +32,10 @@ public class TestInetAddressH2 {
     @RegisterExtension
     public DatabaseExtension<?> dbExtension = H2DatabaseExtension.instance();
 
+    protected String getInetType() {
```

Comment:
we tend to order the code "test methods" -> public / protected / package private / private methods from top to bottom. So this method should stay at the bottom of the class.

Reply:
I wished this could be enforced by Checkstyle. There is no core Checkstyle check to do this afaik (only 3rd party).